### DISCUSSION TITLE
TEI base texts?

### DISCUSSION THREAD

**Discussion by gcelano** [korpling/ANNIS#879] [General]
My base texts are TEI files, which I could theoretically refer to using XPointer in the tokenization file. Is this somehow possible at the moment?

**Comment by thomaskrause** [comment]
That dependens on the implementation of the Pepper PAULA importer. The problem would be, that the base text file still needs to be a PAULA XML file and the pointers only reach inside the `body` tag. Since TEI XML is not valid Paula XML, the Pepper importer will probably fail to parse it.

We are currently working on a Pepper successor that uses graphANNIS internally instead of Salt (<https://github.com/korpling/annatto/>). It lacks an Paula XML importer/exporter, but this is planned feature. It would be possible to add a flag to the importer to determine if the text input file is a Paula XML file and if it should refert to the text in the body, or if the whole XML file should be interpreted as plain text and the offset is based on the characters of the text file.
